package.json

```
@@ -49,7 +49,8 @@
     "eslint-plugin-security": "^1.5.0",
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-sonarjs": "^0.18.0",
-    "eslint-plugin-unicorn": "^45.0.2"
+    "eslint-plugin-unicorn": "^45.0.2",
+    "eslint-plugin-sort-keys-fix": "^1.1.2",
```

Can't have trailing comma in json
```suggestion
    "eslint-plugin-sort-keys-fix": "^1.1.2"
```